tests/scripts/thread-cert/border_router/test_plat_udp_accessiblity.py

```diff
@@ -87,6 +86,38 @@ def test(self):
         for server_aloc in self.nodes[BR1].get_ip6_address(config.ADDRESS_TYPE.ALOC):
             self._test_srp_server(server_aloc, server_port)
 
+        self._testDhcp6ClientAfterReset(BR1, BR1, BR1)
+
+    def _testDhcp6ClientAfterReset(self, server, client, reset_device):
+        DHCP6_PREFIX = '2001::/64'
+
+        # Configure DHCP6 server
```

Comment:
nit:
```suggestion
        # Configure DHCP6 server.
```
And comments following this one.

Reply:
I will ignore it since it's already merged. :+1: